src/Microsoft.AspNet.Mvc.Localization/IHtmlLocalizer.cs

```diff
@@ -19,6 +19,9 @@ public interface IHtmlLocalizer : IStringLocalizer
         /// <returns>A culture-specific <see cref="IHtmlLocalizer"/>.</returns>
         new IHtmlLocalizer WithCulture(CultureInfo culture);
 
+        // Reviewers: I suggest the Html() methods should just return IHtmlContent because the specialized information
+        // in LocalizedHtmlString is unused. Any objections?
```

Comment:
I don't feel strongly about it. If we're going to provide LocalizedHtmlString as an abstraction, then it can be nice for unit testing if that's a factor.